### DISCUSSION TITLE
Welcome to n98-magerun2 Discussions!

### DISCUSSION THREAD

**Discussion by cmuench** [netz98/n98-magerun2#967] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PaperDS** [comment]
Hi, I'm using command:
n98-magerun2 db:dump --include="@trade" export to get sales data to move to a new server which is out of sync. 
It works very well, but I've found a missing a table which maybe should be included in default list?
Table is inventory_shipment_source.
Again not sure how the list of tables has been decided on.

**Reply by cmuench** [reply]
Thanks.
All the tables and groups are defined in the dist config (bundled) within the phar file.

Trade is only a aggregation of other groups.
See here!: https://github.com/netz98/n98-magerun2/blob/66e15d205814005a317e983404be2fe42c1e3e98/config.yaml#L266

Maybe we have to add the table to one of the other groups.

**Reply by PaperDS** [reply]
Thanks for your quick reply. 
Only starting using n98-magerun2 yesterday, so a lot to learn.

I think the table inventory_shipment_source could be included in sales with other shipment or inventory tables?
Table inventory_shipment_source provides data inside my customer orders when I create a shipment to a customer.

**Reply by cmuench** [reply]
That's a good question. The main reason for the sales group is that we can exclude the order data.
Trade is a more general aggregation.
The `shipment_id` shipment source table is related to the `sales_shipmement` table. So I think we should add the table to the sales group. We have already the `inventory_reservation` in this group.
If you want then you can create a PR. Then you are a Magerun Contributor. :-)

**Reply by PaperDS** [reply]
Not sure how to create a PR. A bit above my pay grade at the moment. Still learning git.. :-)

**Reply by cmuench** [reply]
Sounds like a good way to learn something new 💪
